README.md

```diff
@@ -9,7 +9,7 @@ These are some details on what the game was and how it worked under the hood
 - [Docs](https://docs.lu-dev.net)
 - [Object IDs](https://lu-dev.net/objects)
 - [Packets](https://lcdruniverse.org/lu_packets/lu_packets)
-- [Explorer](https://xiphoseer.de/lu-explorer)
+- [Explorer](https://explorer.lu/dashboard)
```

Comment:
That ones gives a basic auth, which isn't particularly helpful either for people who don't know about it. The page moved to <https://lu-dev.net/lu-explorer> when the repo moved